.github/workflows/tests.yml

```diff
@@ -54,31 +54,23 @@ jobs:
           if: ${{ matrix.openssl == 'on' }}
```

Comment:
@Orycterope please, remove this, not used.

Reply:
nice catch !